### DISCUSSION TITLE
Running two instances

### DISCUSSION THREAD

**Discussion by dabeegmon** [inventree/InvenTree#5142] [Q&A]
Greetings

Want to run two instances of InvenTree. 
(If its important - - - - one for electronics and one for my 'pantry/kitchen'.) 

Do I just set up a second instance with a different port number?
Change name - - - other needs?

TIA

**Comment by matmair** [comment]
While you can do this with docker I would consider if using different categories might also be a solution. You can change the exposed port freely. You can expose as many instances as you want but it is not efficient in regards to space, memory or cpu cycle usage.

**Reply by dabeegmon** [reply]
Hmmmmmmmmm - - - not sure what you mean by different categories. 
Would you please elucidate further?

TIA

**Comment by CodingPupper3033** [comment]
How would you be able to change the exposed web port? I've tried editing the .\InvenTree\docker\production\.env file, and that did not seem to work.

**Reply by wolflu05** [reply]
That should work. You need to make sure that you have recreated the containers (`docker-compose down && docker-compose up -d`) and you need to have two separate copies of the `.env` and `docker-compose.yml` file in two different folders for each instance.
